draft-ietf-scitt-architecture.md

```diff
@@ -547,18 +547,17 @@ It indicates the Signed Statement is securing a JSON content type, and identifyi
 
 To register a Signed Statement, the Transparency Service performs the following steps:
 
-1. **Client authentication:** A Client authenticates with the Transparency Service, to Register Signed Statements on behalf of one or more Issuers.
-Authentication and authorization is implementation-specific, and out of scope of the SCITT Architecture.
-1. **Issuer Verification:** The Transparency Service MUST syntactically validate the Issuer's identity Claims, which may be different than the Client identity.
+1. **Client authentication:** A Client authenticates the Transparency Service before registering Signed Statements on behalf of one or more Issuers.
```

Comment:
This is a MAY.  The client does not HAVE to.

Reply:
This is inconsistent with other text in the security consideration section where the assumption is made that TLS-based communication security is made.

If you use TLS then authentication of the TS to the client is mandatory, authentication of the client to the TS is optional.

Reply:
This is a good point.

I think the intention here is that the server authenticates the client, not that the client authenticates the TS

Reply:
Jon has made the change the other way in SCRAPI and thus this is out of sync.

Reply:
Any suggestion on how to improve the text based on the discussion? @OR13 @roywill